argocd/applications/configs/infra-core.yaml

```diff
@@ -36,6 +36,29 @@ api:
     enforceMultiTenancy: "true"
   resources: null
 
+apiv2:
```

Comment:
Should we disable the old API component?

Reply:
Going to do it in the infra-charts after this gets released.